cogeo_mosaic/backends/base.py

```diff
@@ -40,15 +44,17 @@ def point(self, lng: float, lat: float) -> Tuple[str]:
         """Retrieve assets for point."""
 
     @abc.abstractmethod
-    def read(self, *args, **kwargs) -> MosaicJSON:
+    def read(self) -> MosaicJSON:
```

Comment:
At some point I got some linter errors when the base class's `read` had a different function signature than the inherited classes

Reply:
I think the linter errors were because of the `@lru_cache` wrapper